keps/sig-apps/sidecarcontainers.md

```diff
@@ -162,11 +193,29 @@ Init containers would be able to have `sidecar: true` applied to them as it's an
 
 Older Kubelets that don't implement the sidecar logic could have a pod scheduled on them that has the sidecar field. As this field is just an addition to the Container Spec the Kubelet would still be able to schedule the pod, treating the sidecars as if they were just a normal container. This could potentially cause confusion to a user as their pod would not behave in the way they expect, but would avoid pods being unable to schedule.
 
+## Design Details
 
-## Graduation Criteria
-
+### Test Plan
 //TODO
 
+### Graduation Criteria
+#### Alpha -> Beta Graduation
+* Addressed feedback from Alpha testers
+* Thorough E2E and Unit testing in place
```

Comment:
I suggest adding:

* The beta API either supports the important use cases discovered during alpha testing, or has room for further enhancements that would support them.